pkg/apiserver/apiserver_test.go

```diff
@@ -57,8 +57,11 @@ func convert(obj runtime.Object) (runtime.Object, error) {
 }
 
 // This creates fake API versions, similar to api/latest.go.
-const testVersion = "version"
-const newVersion = "version2"
+var testAPIGroup = "test.group"
+var testGroupVersion = unversioned.GroupVersion{Group: testAPIGroup, Version: "version"}
```

Comment:
The tests were running with invalid `GroupVersion` values, since only `v1` is allowed to not have an API group.  This fixes the behavior of the test, but ends up looking a little noisy.  Ideally, we also move the prefix.